src/assets/translations/en-us.yaml

```diff
@@ -2492,7 +2492,7 @@ probe:
     port:
       label: Check Port
       placeholder: e.g. 80
-      placeholderDuex: e.g. 25
+      placeholderDeux: e.g. 25
```

Comment:
I'm even less sure about this. I can't find any particular hinit about it, so i figured, why not ask?